testdata/src/main/java/com/hedvig/app/testdata/feature/embark/builders/PassageBuilder.kt

```diff
@@ -34,7 +34,7 @@ data class PassageBuilder(
         tooltips = tooltip,
         redirects = redirects,
         action = action,
-        api = api?.let { EmbarkStoryQuery.Api3(fragments = EmbarkStoryQuery.Api3.Fragments(it)) },
+        api = api?.let { EmbarkStoryQuery.Api4(fragments = EmbarkStoryQuery.Api4.Fragments(it)) },
```

Comment:
What is this Api4? What is it referring to?

Reply:
If you have multiple fragments in a graphql query the codegen renames them like this, quite annoying :(